app/src/main/java/com/hedvig/app/feature/home/ui/changeaddress/ITPB.kt

```diff
@@ -0,0 +1,105 @@
+package com.hedvig.app.feature.home.ui.changeaddress
+
+import android.content.res.Configuration
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.RowScope
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.wrapContentSize
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material.Button
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.painter.Painter
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.google.android.material.composethemeadapter.MdcTheme
+
+@Composable
+fun ITPB(
```

Comment:
I think we should be careful with naming composables since they are so general in nature. The name should be easily searchable and explain well what they are used for. Perhaps something like `ImageWithTitleAndParagraph`? Maybe we should discuss naming conventions for composables more in depth 🤔

Reply:
I agree that the name is scuffed, which is one of the reasons why this is still a draft PR 😅

ITPB = Illustration, Title, Paragraph, Button

I'm discussing better terminology with Zak on Figma. I'll update it once we get there 🙂

Reply:
That's alright, I made the draft PR so that I could gather this type of feedback 😉

Reply:
Zak suggested Briefing Screen - what do you think about that name?